File: week00/day03/stack.py
stack = []

stack.append(10)
stack.append(20)
stack.append(30)

print(stack[-1])     # 30
print(len(stack))    # 3

value = stack.pop()  # 30 제거 후 반환
print(value)

print(stack[-1])     # 20

if stack:
    print("Stack is not empty")

File: week00/day03/stack.cpp
#include <iostream>
#include <stack>
using namespace std;

int main() {
    stack<int> st;

    st.push(10);
    st.push(20);
    st.push(30);

    cout << st.top() << '\n';   // 30
    cout << st.size() << '\n';  // 3

    st.pop();                   // 30 제거

    cout << st.top() << '\n';   // 20

    if (!st.empty()) {
        cout << "Stack is not empty\n";
    }

    return 0;
}